transformers/gtfs.py

```diff
@@ -85,7 +86,7 @@ def gtfs_operator_line_memory(db_read: MdbxStorage, txn: TXN, generator_defaults
         elif line.responsibility_set_ref_attribute is not None:
             # TODO: ResponsibilitySet to Operator/Authority should be a separate function
             responsibility_set = cast(
-                ResponsibilitySet, db_read.load_object_by_reference(txn, getFakeRef(line.responsibility_set_ref_attribute, ResponsibilitySetRef))
+                ResponsibilitySet, db_read.load_object_by_reference(txn, getFakeRef(line.responsibility_set_ref_attribute, ResponsibilitySetRef,line.version))
```

Comment:
black would have saved the humilation!
```suggestion
                ResponsibilitySet, db_read.load_object_by_reference(txn, getFakeRef(line.responsibility_set_ref_attribute, ResponsibilitySetRef, line.version))
```
But I wonder why we give responsibilityset a version at all.

Reply:
I don't know, but it wants to get a version parameter.